src/IIIFingest/asset.py

```diff
@@ -1,15 +1,22 @@
+from __future__ import annotations
```

Comment:
Just curious, but was this added to resolve an issue with a [forward reference](https://peps.python.org/pep-0563/#forward-references) in one of the type annotations?

I had to go check the docs on this one since I wasn't familiar with it:
- https://docs.python.org/3/library/__future__.html#id1
- https://peps.python.org/pep-0563/

Reply:
Yeah, this was to enable specifying the output as a class from within that class definition. I referenced this StackOverflow post: https://stackoverflow.com/questions/33533148/how-do-i-type-hint-a-method-with-the-type-of-the-enclosing-class. As an alternative, I could have specified the class as a string, but this seemed neater. My understanding is that this will eventually be supported in vanilla python, so there will be a version in which this import will be unnecessary.

Reply:
Completely agree that this is the way to go since it should eventually be supported in vanilla python. The stackoverflow post is helpful.